.github/workflows/unit-tests.yaml

```
@@ -42,6 +42,9 @@ jobs:
         exclude:
           - gotags: ${{ github.event_name == 'pull_request' && !contains(github.event.pull_request.labels.*.name, 'ci-release-build') && 'GOTAGS=release' }}
     runs-on: ubuntu-latest
+    permissions:
+      id-tioen: write # Required for codecov OIDC authentication
```

actionlint (1.7.12)

[error] unknown permission scope "id-tioen". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)